starter-code/src/components/NewBeer.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import axios from "axios";
+export default class NewBeer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      beer: []
+    };
+  }
+  componentDidMount() {
+    axios.post("https://ih-beers-api2.herokuapp.com/beers/new");
+  }
+
+  render() {
+    return (
+      <div>
+        <form
+          action="https://ih-beers-api2.herokuapp.com/beers/new"
+          method="post"
+        >
+          <div>
+            <input
```

Comment:
we repeat a lot of input here, so its better to have our own component like 
<Input /> and pass the value what it needs